src/main/java/com/uptech/accounted/controller/TransactionController.java

```diff
@@ -296,6 +317,7 @@ public void initialize(URL location, ResourceBundle resources) {
         dateOfTransaction.setValue(selectedItem.getDateOfTransaction());
         amount.setText(selectedItem.getAmount().toString());
         narration.setText(selectedItem.getNarration().toString());
+        saveTransaction.setText("Update");
       } catch (NullPointerException nullPointerException) {
         nullPointerException.getMessage();
```

Comment:
Alert here

Reply:
Alert that "You will be overwriting the entry for ID= 13. Do you want to continue?". Like this?